backend/routes/sequence.js

```diff
@@ -29,14 +29,14 @@ const Sequence = require("../models/sequence");
  *
  * @apiParam {String} teamspace Name of teamspace
  * @apiParam {String} model Model ID
- * @apiParam {String} [revId] Revision unique ID
  */
 
 /**
  * @api {get} /:teamspace/:model/revision(/master/head/|/:revId)/sequences/:sequenceId/activities/:activityId Get activity
- * @apiName getSequenceActivityDetail
+ * @apiName getSequenceActivityDetailOld
```

Comment:
don't modify anything regarding activities, it's already updated by santiago on his branch